select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -53,7 +55,6 @@
 	private String initSelection;  //TODO this will not work
 	private String query;
 	private String width;
-	private String theme;
```

Comment:
I still fail to see how it will break it the only thing I think we don't cover is

https://select2.github.io/examples.html#templating